modules/ROOT/pages/deployment/container/orchestration/orchestration.adoc

```diff
@@ -352,6 +354,44 @@ include::example$deployment/container/orchestration/values-overwrite-tab-3.yaml[
 endif::[]
 ====
```

Comment:
```suggestion
==== Enable Metrics with Prometheus
```
As "metrics" is not a product name with all lower letters but data, we have to capitalize the first letter.